packages/utils/src/logger/logger.ts

```diff
@@ -27,8 +27,8 @@ export const setupConfigLogger = () => {
 }
 
 export abstract class Logger {
-  static getLogger(category?: string, useDebug = true): ConfigLogger | DebugLogger {
-    return useDebug ? new DebugLogger(category.replace('.', ':')) : new ConfigLogger(category)
+  static getLogger(category?: string, useDebug = true): Logger {
```

Comment:
Just out of curiosity, was `ConfigLogger | DebugLogger` working?

Reply:
Yeah!